### PR TITLE
Add "auto" theme option to theme picker dropdown

### PR DESCRIPTION
**Pull request by shravanngoswamii**
"auto" theme is added in theme picker, is it ok?
https://github.com/JuliaDocs/Documenter.jl/pull/2077#discussion_r1145651759

### CODE REVIEW

**Comment by mortenpi** [comment]
Yeah, this looks like what I had in mind. But it also needs to have the corresponding entry in the dropdown for this to work.

Also, please don't open separate PRs, but iterate on the code in the existing PR.

**Comment by shravanngoswamii** [comment]
Entry in dropdown is added, will it work now!!

File: assets/html/js/themes.js [hunk 1]
```diff
@@ -34,10 +34,20 @@ function set_theme(theme) {
 
 // Theme picker setup
 $(document).ready(function() {
+  // Add "auto" option to the dropdown
+  $('#documenter-themepicker').append($('<option>', { 
+    value: 'auto',
+    text : 'Auto'
```
Comment (mortenpi) [inline assets/html/js/themes.js:40]:
```suggestion
    text : 'Automatic (OS preference)' 
```

**Review by mortenpi — COMMENTED** [review]
A few more things:

* It should also update the theme if you change to `auto`. I.e. basically needs a call to `set_theme`, but might need to be updated a bit to handle this case.
* It would be great to have a small note in `CHANGELOG.md` about this as well.

**Comment by mortenpi** [comment]
The switching back to OS theme doesn't seem to work. Does it work for you when you run it locally?

**Comment by shravanngoswamii** [comment]
Actually, I am having an error while running Documenter.jl locally, i update the code soon if OS theme doesn't work!!

**Pull request closed without merging**